packages/devops/scripts/lambda/actions/spin_up_tupaia_deployment.py

```diff
@@ -71,11 +71,20 @@ def spin_up_tupaia_deployment(event):
         delete_after = datetime.now() + timedelta(hours=event['HoursOfLife'])
         extra_tags.append({ 'Key': 'DeleteAfter', 'Value': format(delete_after, "%Y-%m-%d %H:%M") })
 
-    if 'StartAtUTC' in event:
-        extra_tags.append({ 'Key': 'StartAtUTC', 'Value': event['StartAtUTC'] })
+    if deployment_name == 'production':
+        if 'StartAtUTC' in event or 'StopAtUTC' in event:
+            raise Exception('Production deployment cannot have StartAtUTC/StopAtUTC')
+    else:
+        if 'StartAtUTC' in event:
+            extra_tags.append({ 'Key': 'StartAtUTC', 'Value': event['StartAtUTC'] })
```

Comment:
Wondering if we should check for a valid start/stop time here? Like would it cause issues later when trying to read these tags if invalid time/empty string entered by accident? No biggy as this is how it was working previously.

Reply:
Yeah good point. Maybe in the future if we run across any issues, it's mostly automatically set